### ISSUE TITLE
[Bug]: executing the sparse_vector_search.py script directly from the example, an error occurred

### ISSUE THREAD

**Issue by warran2** [infiniflow/infinity#1706]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Version or Commit ID

v0.3.0-dev6

### Other environment information

```Markdown
ubuntu22
python3.10.12
```

### Actual behavior and How to reproduce it

root@WIN-CBK2VJCUKE4:/home/warran/infinity/example# python3 sparse_vector_search.py
(<ErrorCode.INVALID_EXPRESSION: 3069>, "Invalid constant type: <class 'dict'>")

**Comment by warran2** [comment]
If I remove the try... except... the error output is:

root@WIN-CBK2VJCUKE4:/home/warran/infinity/example# python3 sparse_vector_search.py
Traceback (most recent call last):
  File "/home/warran/infinity/example/sparse_vector_search.py", line 42, in <module>
    table_instance.insert(
  File "/usr/local/lib/python3.10/dist-packages/infinity/local_infinity/table.py", line 161, in insert
    constant_expression = get_local_constant_expr_from_python_value(value)
  File "/usr/local/lib/python3.10/dist-packages/infinity/local_infinity/utils.py", line 207, in get_local_constant_expr_from_python_value
    raise InfinityException(ErrorCode.INVALID_EXPRESSION, f"Invalid constant type: {type(value)}")
infinity.common.InfinityException: (<ErrorCode.INVALID_EXPRESSION: 3069>, "Invalid constant type: <class 'dict'>")

**Comment by JinHai-CN** [comment]
The API is updated, but the examples are not updated. Use following code:

```

import infinity
from infinity.common import SparseVector

try:
    # open a local directory to store the data
    infinity_instance = infinity.connect("/var/infinity")

    # connect to server with 127.0.0.1
    # infinity_instance = infinity.connect(infinity.common.LOCAL_HOST)

    # 'default_db' is the default database
    db_instance = infinity_instance.get_database("default_db")

    # Drop my_table if it already exists
    db_instance.drop_table("my_table", infinity.common.ConflictType.Ignore)

    # Create a table named "my_table"
    table_instance = db_instance.create_table("my_table", {
        "num": {"type": "integer"},
        "body": {"type": "varchar"},
        "vec": {"type": "sparse,100,float,int"}
    })

    # Insert 3 rows of data into the 'my_table'
    table_instance.insert(
        [
            {
                "num": 1,
                "body": r"unnecessary and harmful",
                "vec":  SparseVector([10, 20, 30], [1.1, 2.2, 3.3])
            },
            {
                "num": 2,
                "body": r"Office for Harmful Blooms",
                "vec": SparseVector([40, 50, 60], [4.4, 5.5, 6.6])
            },
            {
                "num": 3,
                "body": r"A Bloom filter is a space-efficient probabilistic data structure, conceived by Burton Howard Bloom in 1970, that is used to test whether an element is a member of a set.",
                "vec":  SparseVector([70, 80, 90], [7.7, 8.8, 9.9])
            },
        ]
    )

    result = table_instance.output(["num", "vec", "_similarity"]).match_sparse("vec", SparseVector([0, 20, 80], [1.0, 2.0, 3.0]), "ip", 3).to_pl()
    print(result)
    infinity_instance.disconnect()

except Exception as e:
    print(str(e))

```

**Comment by JinHai-CN** [comment]
Fixed by #1710
